src/main/java/com/nexmo/quickstart/Util.java

```diff
@@ -58,4 +59,29 @@ public static String envVar(String key) {
         }
         return value;
     }
+
+    /**
+     * Look up a required environment variable and throw an
+     * IllegalArgumentException if the value is not one of "0", "1", "true", "false", "on" or "off".
```

Comment:
Can't say it was a conscious decision. I don't think either method hurts though.